src/Connections/GitlabRepo.ts

```diff
@@ -128,11 +144,23 @@ export class GitLabRepoConnection extends CommandConnection {
     static helpMessage: (cmdPrefix?: string | undefined) => MatrixMessageContent;
     static ServiceCategory = "gitlab";
 
+	static validateState(state: Record<string, unknown>, isExistingState = false): GitLabRepoConnectionState {
+        const validator = new Ajv({ strict: false }).compile(ConnectionStateSchema);
+        if (validator(state)) {
+            // Validate ignoreHooks IF this is an incoming update (we can be less strict for existing state)
```

Comment:
In this file this is already enforced by the JSON schema. Is this bit copypasted, or is the schema unnecessarily strict?

Reply:
Think I just failed to commit this actually.